<launch>

  <group ns="texting">
    
    <node pkg="multiprocess_render" type="user_input.py" name="render1">
      <param name="sink" value="/render1/text1" />

  

    </node>

  </group>
<group ns="storing">
    
    <node pkg="multiprocess_render" type="file_store.py" name="store1">
      <param name="source" value="/render1/text1" />
      <param name="sink" value="/render1/text2" />
      
   

    </node>

  </group>
<group ns="reading">
    
    <node pkg="multiprocess_render" type="robot_answer.py" name="answer1">
      <param name="source1" value="/render1/text2" />
      <param name="source2" value="/render1/index" />
      
   

    </node>

  </group>
<group ns="displaying">
    
    <node pkg="multiprocess_render" type="display_answer.py" name="display1" output="screen">
      <param name="sink" value="/render1/index" />
      <param name="source1" value="/render1/show1" />
      <param name="source2" value="/render1/show2" />
      <param name="source3" value="/render1/show3" />
      <param name="source4" value="/render1/show4" />
      <param name="source5" value="/render1/show5" />
      <param name="sink" value="/render1/index" />

     

    </node>

  </group>

</launch>
